src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -729,6 +734,20 @@ private PrecompileArguments GetPrecompileArgumentsFor(ITaskItem assemblyItem, st
             }
         }
 
+        if ((parsedAotMode == MonoAotMode.Full ||
```

Comment:
Do you need to check against all of the modes? Isn't it enough if the user asked for the EnableUnmanagedCallersOnlyMethodsExports? If you run compiler in a mode that won't generate any output there will be no output file or a file without any exported methods.

Reply:
I guess we dont have to check, but should a warning be generated instead if `EnableUnmanagedCallersOnlyMethodsExports` is toggled when not in full aot mode (any of these modes) to warn the user that it is only useful in full aot mode? Maybe another block in `ProcessAndValidateArguments`?

Reply:
Just check if you get an exports file back and if not, you could emit a warning, since then the user has called the task with an incorrect combination of arguments.

Reply:
Added a check for export symbols after compiling assemblies.